swarm/pss/client/client_test.go

```diff
@@ -228,9 +228,13 @@ func newServices() adapters.Services {
 			defer cancel()
 			keys, err := wapi.NewKeyPair(ctxlocal)
 			privkey, err := w.GetPrivateKey(keys)
-			psparams := pss.NewPssParams(privkey)
+			psparams := pss.NewPssParams()
+			psparams.Init(privkey)
```

Comment:
why not:
```
psparams := pss.NewPssParams().WithPrivateKey(pk)
// then below instead of init:
func (self *PssParams) WithPrivateKey(privatekey *ecdsa.PrivateKey) *PssParams {
	self.privateKey = privatekey
        return self
}
```

Reply:
Did we settle on this policy? If so, I can make the change.